packages/web/src/pages/user/[[...uid]].tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { MarketingLayout } from '../../components/Layout';
+import { useRouter } from 'next/router';
+import { UserProfileLayout, UserProfileData } from '../../components/UserProfile/UserProfileLayout';
+import { Alert, AlertIcon } from '@chakra-ui/alert';
+export { getServerSideProps } from '../../components/Chakra';
+
+const UserProfile: NextPage = () => {
+  const router = useRouter() || { query: { text: '' } };
+  const { uid } = router.query;
+
+  const [user, setUser] = React.useState<UserProfileData>();
```

Comment:
We don't want to rely on the properties from another component to type our User here. Since the component is only showing what it needs, this should be the type that comes back from the server. So I would create a new user interface after the imports and use that here. (It may have the same values, but that could change in the future. The component may require more than what is coming back from the API and we want to be flexible if that happens)

```suggestion
  const [user, setUser] = React.useState<User>();
```